package.json

```diff
@@ -206,9 +201,6 @@
         },
         "build:css": {
             "command": "node ./tasks/build-css.js",
```

Comment:
Still want to run this script because it converts CSS into the a wrapped typescript version.